### PR TITLE
Add "Larger Circle" mod

### PR DESCRIPTION
**Pull request by ojh050118**
For mobile device

### CODE REVIEW

File: osu.Game/Rulesets/Mods/ModLargerCircle.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        public override string Acronym => "LC";
+        public override IconUsage Icon => FontAwesome.Solid.ChevronCircleUp;
+        public override ModType Type => ModType.DifficultyReduction;
+        public override double ScoreMultiplier => 1.0;
+        public override bool Ranked => true;
+        public override Type[] IncompatibleMods => new[] { typeof(ModHardRock) };
+
+        public void ApplyToDifficulty(BeatmapDifficulty difficulty)
+        {
+            const int ratio = 1;
```
Comment (peppy) [inline osu.Game/Rulesets/Mods/ModLargerCircle.cs]: `1`

Comment (ojh050118) [reply osu.Game/Rulesets/Mods/ModLargerCircle.cs]: Ah ... sorry I made a mistake.

**Comment by ojh050118** [comment]
I'll recreate the PR.

**Comment by bdach** [comment]
I don't think there's a need to recreate the PR, you can just commit your fixes and then push again.

You should however make sure the build passes if you want to get this merged, yours failed on code inspections. [Here are the instructions on running the code inspections locally](https://github.com/ppy/osu#code-analysis), it's a good practice to do so before pushing.

**Comment by ojh050118** [comment]
I cannot open the pull request again because the repository has been deleted.

**Comment by ojh050118** [comment]
closed #5993 via #6007

**Comment by bdach** [comment]
> I cannot open the pull request again because the repository has been deleted.

I assume you deleted your fork? This is not how forks should be kept up-to-date. For future reference, please see [*Syncing a fork*](https://help.github.com/en/articles/syncing-a-fork) for detailed instructions.

**Pull request closed without merging**